src/components/Editor/MediaHandler.vue

```diff
@@ -28,13 +28,12 @@
 		@dragover.prevent.stop="setDraggedOver(true)"
 		@dragleave.prevent.stop="setDraggedOver(false)"
 		@file-drop="onEditorDrop">
-		<input ref="attachmentFileInput"
-			tabindex="-1"
+		<input v-show="false"
+			ref="attachmentFileInput"
 			data-text-el="attachment-file-input"
 			type="file"
 			accept="*/*"
 			aria-hidden="true"
```

Comment:
@ShGKme Also do we still need the aria-hidden then or is this already fine with just having display: none; through v-show

Reply:
No, `display: none` (and `visibility: hidden`) both make element completely hidden.

Let's say, be default element is visible for both monitor's and screen reader's users.

- `display: none`/`visibility: hidden` - hides element completely for everything
- `aria-hidden="true"` - hides element only for screen readers (meaning, it is not important element, like decorative image)
- `hidden-visually` - hides element only visually (meaning, this is only for screen reader or replaces some only visual element)

One specific case when `aria-hidden="true"` and `hidden-visually` used together - the prev and next slides in Viewer. They must be hidden, but for image/video pre-loading and because of a bug in FF we need then to not be `display: none`.

Reply:
Thanks, that makes sense ❤️